handbook/engineering/hiring/software-engineer-coding-exercise.md

```diff
@@ -65,6 +65,19 @@ We designed [this synchronous exercise](https://github.com/sourcegraph/interview
 
 If we device to move forward, we will schedule interviews with team members.
 
+## Kubernetes exercise
+
+We designed this exercise to measure your skills at integrating build and deployment workflows into a development lifecycle.
+
+- You will choose a two hour timeframe to independently work on the exercise. You can choose any day and time that works for you; there are no scheduling constraints.
+- At your chosen date and time, we will send you detailed instructions. You will be building a simple pipeline to build and deploy a set of Go services.
+- You should have a recent version of Go (≥ 1.13) installed on your computer.
+- You should have a simple Kubernetes local cluster ([kind](https://kind.sigs.k8s.io/), [k3d](https://github.com/rancher/k3d), [minikube](https://minikube.sigs.k8s.io/docs/start/))
```

Comment:
Did you verify that the yaml works on all of these? PVs behave differently, which may impact the solution.

Reply:
I tested `kind` and `k3d`, but minikube should also work as I believe it has a stateful volume. Ill merge this as I have an email pending on it and make sure to verify and update before I send the actual test.

Reply:
- [x] `kind`
- [x] `k3d`
- [ ] `minikube`